command/clouds/aws/cloudfront.go

```diff
@@ -1,105 +0,0 @@
-package aws
-
-import (
-	"fmt"
-	"strconv"
-	"strings"
-	"time"
-
-	"github.com/aws/aws-sdk-go/service/cloudfront"
-	"github.com/innogames/slack-bot/v2/bot"
-	"github.com/innogames/slack-bot/v2/bot/config"
-	"github.com/innogames/slack-bot/v2/bot/matcher"
-	"github.com/innogames/slack-bot/v2/bot/msg"
-	"github.com/innogames/slack-bot/v2/client"
-	"github.com/slack-go/slack"
-)
-
-// command to trigger/start jenkins jobs
-type cloudFrontCommand struct {
-	awsCommand
-	service *cloudfront.CloudFront
-	cfg     []config.AwsCfDistribution
-}
-
-// NewAwsCommand is a command to interact with aws resources
-func newCloudFrontCommands(cfg []config.AwsCfDistribution, base awsCommand) bot.Command {
-	svc := cloudfront.New(base.session)
-	return &cloudFrontCommand{base, svc, cfg}
-}
-
-func (c *cloudFrontCommand) GetMatcher() matcher.Matcher {
-	return matcher.NewGroupMatcher(
-		matcher.NewTextMatcher("aws cf list", c.distributions),
-		matcher.NewRegexpMatcher(`aws cf clean (?P<DIST>[\w|\d]+) at (?P<PATH>[\w|\d|\W]+)`, c.clearCache),
-	)
-}
-
-func (c *cloudFrontCommand) distributions(match matcher.Result, message msg.Message) {
-	// show list
-	blocks := []slack.Block{}
-	for _, v := range c.cfg {
-		txtObj := slack.NewTextBlockObject("mrkdwn", fmt.Sprintf("\"%s\": %s\n", v.ID, v.Name), false, false)
-		if err := txtObj.Validate(); err != nil {
-			fmt.Println(err.Error())
```

Comment:
I'd recommend to use the logger instead of fmt

Reply:
I've removed `command/clouds/aws/cloudfront.go` file. So there is no logging what using fmt.